style/post.css

```diff
@@ -15,6 +15,10 @@
   visibility: visible;
 }
 
+.post-page footer.mobileFooter {
```

Comment:
I'd personally prefer a "lowercase with dashes" notation for CSS classes (`mobile-footer`), but I'm fine either way.

Reply:
sure, can do that